### PR TITLE
Expanded the Page Params section in variables.md

### PR DESCRIPTION
**Pull request by renatovargas**
Expanded on the use of ##Page Params in the templates/variables.md
documentation. Added sample code for something that keeps coming up on
discuss.github.io

### CODE REVIEW

**Comment by digitalcraftsman** [comment]
Hello @renatovargas,

great to see that everything worked as expected. Your provided example is a good extension for the docs to clarify the usage of frontmatter variables in combination with templates.

Could you accept the CLA as final step?

In your commit message I think you mean `discuss.gohugo.io` instead of `discuss.github.io`, right?

**Comment by renatovargas** [comment]
Hello @digitalcraftsman, 

I do mean `discuss.gohugo.io`. My bad. 

I already accepted the CLA, but I don't know why it doesn't update here.

Thanks again.

**Comment by renatovargas** [comment]
Didn't meant to close the pull request. My mom jokes that the first 20 years at anything are the most difficult.

**Comment by digitalcraftsman** [comment]
No problem. Maybe I found the problem with the CLA. Github identifies users with the help of their email addresses. The comment of the CLA linked [this Github help page](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user) to explain the issue.

As discribed there, you'll find a question mark next to your username at the [commit page](https://github.com/spf13/hugo/commit/c1197d9c66366580ab9c6a9b982c2374a78d9513). When hovering over it the notification suggests to add your email address in the email settings.

In conclusion: your commit contains an email address that is not associated with your Github account. Therefore, this commit isn't treated as "yours".

**Comment by renatovargas** [comment]
You were right, in order to prevent spam issues I added a fake email address which would show publicly with my commits but hadn't added it to GitHub..

**Comment by digitalcraftsman** [comment]
Merged as 817d69487cbe58d38b8b13312c203fba42817bcf (master) and df2ac4d39918cb481d0a3c14adcc994961dfda3b (v0.15.docs). Your additions should go live in a few minutes.

Hopefully, the next time you contribute we will have less trouble :smile:

**Comment by renatovargas** [comment]
Cool!
